### DISCUSSION TITLE
How to use an external CSL for reference formatting?

### DISCUSSION THREAD

**Discussion by tillgrallert** [alshedivat/al-folio#941] [Q&A] [answered]
I am trying to use a CSL style from an external URL in order to format my publications. I did provide the URL in `_config.yml` like so

```
style: https://raw.githubusercontent.com/tillgrallert/CSL/master/chicago-fullnote-bibliography-with-ibid.csl
```

But this throws the following error in the during build and deployment, even though the [file](https://raw.githubusercontent.com/tillgrallert/CSL/master/chicago-fullnote-bibliography-with-ibid.csl) is readily available:

```
No such file or directory @ rb_sysopen - https://raw.githubusercontent.com/tillgrallert/CSL/master/chicago-fullnote-bibliography-with-ibid.csl (Errno::ENOENT)
```

**Comment by rohandebsarkar** [comment, marked as answer]
Publication formatting is done using the `jekyll-scholar` plugin. Please refer to their documentation: https://github.com/inukshuk/jekyll-scholar#configuration

By looking at the error, it seems that it is trying to open a local file. So you may try downloading the file and then use the local file.
